packages/builders/scripts/vats/upgradeVaults.js

```diff
@@ -0,0 +1,23 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
+
+/** @type {import('@agoric/deploy-script-support/src/externalTypes.js').ProposalBuilder} */
+export const defaultProposalBuilder = async ({ publishRef, install }) =>
+  harden({
+    sourceSpec: '@agoric/inter-protocol/src/proposals/upgrade-vaults.js',
```

Comment:
> It's annoying to have multiple files with the same name.

Like `package.json`, `types.js` and `typeGuards.js`? :-P

I think it's more confusing that it's not clear which `proposal` maps to which `script`. Maybe we should move to having a single module that provides both aspects; they're rarely separate.

Regardless, we should avoid camelCase in filenames because of the problem for case insensitive file systems. But I won't block on it.